src/tests/Dictionary/DictionaryContainer.test.jsx

```diff
@@ -155,7 +155,7 @@ describe('DictionaryOverview', () => {
     expect(spy).toHaveBeenCalledTimes(1);
   });
 
-  it('should handle componentWillUpdate in HEAD version release', () => {
+  it('should fetch all released versions when a new collection version is released', () => {
```

Comment:
What has this got to do with the invalid date?

Reply:
I had to re-write this test and test the changed functionality.

Reply:
I still do not understand how it is related to a ticket which is dealing with the display of an invalid date. The only test that i would expect in such a ticket would be "it should display a valid date...."

Reply:
The root cause of the 'invalid date' bug was that the released collections' data was not being updated (re-fetched from the backend) after a new release. That's what I actually changed and is what this is testing for. Displaying 'invalid date' was a result of the data not being fetched.